webconsole/v1/types.go

```diff
@@ -28,40 +43,21 @@ type WebConsoleConfiguration struct {
 
 	// MetricsPublicURL is the public endpoint for metrics (optional)
 	MetricsPublicURL string `json:"metricsPublicURL" protobuf:"bytes,6,opt,name=metricsPublicURL"`
+}
 
-	// ExtensionScripts are file paths on the asset server files to load as scripts when the Web
-	// Console loads
-	ExtensionScripts []string `json:"extensionScripts" protobuf:"bytes,7,rep,name=extensionScripts"`
-
-	// ExtensionProperties are key(string) and value(string) pairs that will be injected into the console under
-	// the global variable OPENSHIFT_EXTENSION_PROPERTIES
-	ExtensionProperties map[string]string `json:"extensionProperties" protobuf:"bytes,8,rep,name=extensionProperties"`
-
-	// ExtensionStylesheets are file paths on the asset server files to load as stylesheets when
-	// the Web Console loads
-	ExtensionStylesheets []string `json:"extensionStylesheets" protobuf:"bytes,9,rep,name=extensionStylesheets"`
-
-	// Extensions are files to serve from the asset server filesystem under a subcontext
-	Extensions []AssetExtensionsConfig `json:"extensions" protobuf:"bytes,10,rep,name=extensions"`
+// ExtensionsConfiguration holds custom script, stylesheets, and properties used for web console customization
+type ExtensionsConfiguration struct {
+	// ExtensionScriptURLs are URLs to load as scripts when the Web Console loads. The URLs must be accessible from
+	// the browser.
+	ExtensionScriptURLs []string `json:"extensionScriptURLs" protobuf:"bytes,1,rep,name=extensionScriptURLs"`
```

Comment:
Interesting.  No need for a map anymore?

Reply:
There's still extension properties, but the old `extensions` stanza from asset config is no longer supported